faq.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Frequently Asked Questions (FAQ)
+---
+
+- this list will be replaced by the Table of Contents
+{:toc}
+
+## Installation
+
+### How do I install a recent version of Cockpit on Debian or Ubuntu?
+
+While Debian and Ubuntu both ship with access to Cockpit in their main repositories, the version in each might be older than the latest available. This is due to the nature of "stable" and "long term support" distributions.
+
+There's often a newer version of Cockpit available for both Debian and Ubuntu, in an extra "backports" repository. We recommend using this newer version, if you're able to do so.
+
+Please read the installation instructions for the appropriate distribution:
+- [Debian installation instructions](running.html#debian)
+- [Ubuntu installation instructions](running.html#ubuntu)
+
+### How do I use Cockpit with a proxy?
+
+Cockpit can be used from behind a proxy.
+
+Read one of the following documents:
+
+- [Proxying Cockpit over NGINX](external/wiki/Proxying-Cockpit-over-NGINX.html)
+- [Proxying Cockpit over Apache with LetsEncrypt](external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt.html)
+
+## Development
+
+### How do I build an app for Cockpit?
+
+- [Starter Kiit](blog/cockpit-starter-kit.html)
+- [Make your Cockpit page easily installable](blog/making-a-cockpit-application.html)
+- [Additional developer-related resourcesare on the Contributing page](external/wiki/Contributing.html)
+
+## Troubleshooting
+
+### Login
+
+#### Blank white page after login
```

Comment:
If this happens, it's always a good first step to check both the browser console (Ctrl+Shift+J in most browsers) as well as the system logs (e.g. `sudo journalctl --since "5 minutes ago"`) for related error messages.